invoicer_project/static/js/login_request.js

```diff
@@ -0,0 +1,105 @@
+const host = "http://127.0.0.1:8000";
+const emailForm = document.getElementById("email_input_log_in_page");
+const passwordForm = document.getElementById("password_input_log_in_page");
+
+
+function validatePassword() {
+    return passwordForm.value !== '';
+}
+
+function validateEmail() {
+    let user_email = emailForm.value;
+    return !(user_email.includes(' ') || !(/^[a-zA-Z0-9.]{3,20}@(?:[a-zA-Z0-9]{2,20}\.){1,30}[a-zA-Z]{2,10}$/.test(user_email))) 
+}
+
+function validateLoginDataOnFrontEnd() {
+    const emailRes = validateEmail();
+    const passwordRes = validatePassword();
```

Comment:
passwordRes to passwordInputValidationResult
or
something like pswrdInpValRes

Reply:
Max said to keep id's and variable's names clear and understandable, but still short, so I don't see point in that. That block of code is pretty simple and obvious, so there is no reason for changing names of variables, especially when the naming style and correct and the code works